packages/kit/types/index.d.ts

```diff
@@ -80,6 +80,11 @@ export interface Builder {
 			replace?: Record<string, string>;
 		}
 	): string[];
+
+	/**
+	 * @param {string} directory A path to file that would be compress.
```

Comment:
```suggestion
	 * @param {string} directory Path to the directory containing the files to be compressed
```

Reply:
It's clearer in your way. I will add it.